src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return sum - 20;
+  } else if (days === 7) {
+    return sum - 50;
+  }
```

Comment:
There's a logical error in the discount application. According to the task, if you rent the car for 7 or more days, you should get $50 off your total. However, the current condition only applies the discount exactly on the 7th day.